crates/core/c8y_api/src/smartrest/operations.rs

```diff
@@ -119,6 +147,55 @@ impl Operation {
             .map(|exec| exec.forceful_timeout)
             .unwrap_or(DEFAULT_FORCEFUL_TIMEOUT)
     }
+
+    pub fn is_valid_opeation_handler(&self) -> bool {
+        if self.exec.is_none() {
+            return true; // can be an empty operation file, not meant for custom operation handler
```

Comment:
Returning false makes more sense: this is not an operation handler.

Reply:
Yes, but `CumulocityConverter::operations` needs to contain **both** custom operation handlers and empty operation declaration files. 
https://github.com/thin-edge/thin-edge.io/blob/main/crates/extensions/c8y_mapper_ext/src/converter.rs#L1425

I have an idea. I'll create another function, `is_supported_operation_file()`, then I'll change this like https://github.com/thin-edge/thin-edge.io/blob/f9ef26492cd7f7b264ce652b4998c626f4b1926c/crates/core/c8y_api/src/smartrest/operations.rs#L306-L308 to

```rust
            if details.is_valid_opeation_handler() || details.is_supported_operation_file() {
                operations.add_operation(details);
            }
```

Reply:
Done in 31913a47803f14de3bb1d2c6c5089cf223f3df14